### PR TITLE
fix: remove reify

### LINKED ISSUE #33: Get rid of reify
Please get rid of reify, and set up babel, because any module importing `gramps-cli` is forced to add `require('reify')`, otherwise the code cannot be imported. See https://github.com/supergraphql/graphql-cli-gramps/blob/master/src/index.ts

### PR DESCRIPTION
**Pull request by jlengstorf**
close #33

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,14 +18,18 @@
...
   "bin": {
-    "gramps": "bin/index.js"
+    "gramps": "gramps.js"
   },
   "scripts": {
-    "gramps": "bin/index.js",
+    "gramps": "./gramps.js",
     "prepush": "npm test",
-    "lint": "eslint {bin, gateway, lib, test}/**/*.js",
+    "build": "babel src -d .",
```
Comment (ecwyne) [inline package.json]: Should this be `babel src -d dist` to put the built artifact in a separate directory? It's a bit confusing to have a `gateway/`, `lib/`, `gramps.js` added to the root of the repo after build.

Comment (jlengstorf) [reply package.json]: I put them in the root to make it easier for anyone looking at their `node_modules`. I don't have a particularly strong preference, but I always like it when I open up a package and have the main files at `/node_modules/packagename/index.js`.

Comment (ecwyne) [reply package.json]: https://github.com/apollographql/graphql-tools/blob/master/package.json#L5
https://github.com/facebook/dataloader/blob/master/package.json#L68
https://github.com/petkaantonov/bluebird/blob/master/package.json#L69

It looks like most packages have a dist/release directory. I looked at these three and also `recompose` which I couldn't immediately figure out to be honest.

interestingly it looks like `dataloader` has an `index.js` at it's root that just calls into `dist/`

Comment (jlengstorf) [reply package.json]: Okay, cool. I moved them to `dist` and updated the `.gitignore` to match.

**Pull request merged**